sorc/link_fv3gfs.sh

```diff
@@ -35,11 +35,11 @@ if [ $machine == "cray" ]; then
 elif [ $machine = "dell" ]; then
     FIX_DIR="/gpfs/dell2/emc/modeling/noscrub/emc.glopara/git/fv3gfs/fix_nco_gfsv16.3.0"
 elif [ $machine = "wcoss2" ]; then
-    FIX_DIR="/lfs/h2/emc/global/save/emc.global/FIX/fix_nco_gfsv16.3.0"
+    FIX_DIR="/lfs/h2/emc/global/noscrub/emc.global/FIX/fix_nco_gfsv16.3.23"
```

Comment:
Is there an additional fix file update to 16.3.23? Or should this be the 16.3.22 fix files?

Reply:
You're correct, I got them confused. This should still be the v16.3.22 set that had updated buoy files for the GoM name change. I'll revert this back, thanks @JessicaMeixner-NOAA !

Reply:
Reverted back @ 861b568e0